indexer/scripts/elastic-conf.py

```diff
@@ -0,0 +1,106 @@
+"""
+configure Elasticsearch
+"""
+
+import argparse
+import json
+import subprocess
+import time
+from collections import Counter
+from logging import getLogger
+from typing import Any, Dict, List, cast
+
+from elastic_transport import ConnectionError, ConnectionTimeout
+from elasticsearch import Elasticsearch
+
+from indexer.app import App
+from indexer.elastic import ElasticMixin
+
+logger = getLogger("elastic-stats")
+
+
+class ElasticConf(ElasticMixin, App):
+    def main_loop(self) -> None:
+        es = self.elasticsearch_client()
+        assert es.ping(), "Failed to connect to Elasticsearch"
+
+        index_template_path = "./conf/elasticsearch/create_index_template.json"
+        ilm_policy_path = "./conf/elasticsearch/create_ilm_policy.json"
+        initial_index_template = "./conf/elasticsearch/create_initial_index.json"
+        # snapshot_policy_path = "/elasticsearch/conf/create_snapshot_policy.json"
+
+        index_template_created = self.create_index_template(es, index_template_path)
+        ilm_policy_created = self.create_ilm_policy(es, ilm_policy_path)
+        alias_created = self.create_initial_index(es, initial_index_template)
+        # snapshot_policy_created = self.create_snapshot_policy(es, snapshot_policy_path)
+
+        if index_template_created and ilm_policy_created and alias_created:
+            logger.info("All ES configurations applied successfully.")
+        else:
+            logger.error("One or more configurations failed. Check logs for details.")
+
+    def create_index_template(self, es: Elasticsearch, file_path: str) -> bool:
+        with open(file_path, "r") as file:
+            data = file.read()
+
+        json_data = json.loads(data)
```

Comment:
These are repeated in every method... may be there should be just one private method to load JSON from file path?